release-notes/opensearch-release-notes-2.12.0.md

```diff
@@ -0,0 +1,1306 @@
+<li>Concurrent segment search is now generally available, giving you the option to query index segments in parallel at the shard level. This can deliver improved latency for many types of search queries.</li>
+<li>Date histogram aggregations without sub-aggregations can now be transformed into and executed as range filters, offering a significant boost to search performance.</li>
+<li>Multi-terms aggregations are now significantly faster for high-cardinality search terms, which offers improved performance for many prefix and wildcard queries.</li>
+<li>A new match-only text field type, a variant of the text field, can help reduce storage costs while maintaining term and multi-term query performance.</li>
+<li>The keyword, numeric, and IP field types can now be searched with doc_values queries. This type of query can decrease storage requirements for rarely accessed fields.</li>
+<li>A new top N queries feature lets you track high-latency queries with an API.</li>
+<li>Updates to the Discover tool in OpenSearch Dashboards include improvements to density, column order, sorting controls, and more. Users can now choose between the previously implemented Discover experience and the updated experience.</li>
+</ul>
+
+<h2>EXPERIMENTAL FEATURES</h2>
+<ul>
+<li>OpenSearch 2.12.0 includes the following experimental features. Experimental features are disabled by default. For instructions on how to enable them, refer to the documentation for the feature.</li>
+<li>The OpenSearch Assistant Toolkit helps developers build generative AI experiences inside of OpenSearch Dashboards. With integrated natural language processing and context-aware features, developers can use this toolkit to apply generative AI to create interactive user experiences and extract insights from OpenSearch data.</li>
+<li>A new agent framework added to ML Commons uses remote large language learning models (LLMs) for step-by-step problem-solving and can coordinate machine learning tools using LLMs. The framework includes a flow agent and a conversational agent in this release.</li>
+<li>Users can now query multiple clusters with cross-cluster monitors through the Alerting plugin.</li>
+</ul>
+
+<h2>DEPRECATION NOTICE</h2>
+
+<h3>CentOS7</h3>
+<ul>
+Please note that OpenSearch will deprecate support for CentOS Linux 7 as a continuous integration build image and supported operating system in an upcoming release, as CentOS Linux 7 will reach end-of-life on June 30, 2024 (see <a href="https://blog.centos.org/2023/04/end-dates-are-coming-for-centos-stream-8-and-centos-linux-7/">#notice</a>). To view OpenSearch's compatible operating systems, visit <a href="https://blog.centos.org/2023/04/end-dates-are-coming-for-centos-stream-8-and-centos-linux-7/">#operating-system-compatibility</a>.
```

Comment:
the first link looks right, but the second link should be replaced with: https://opensearch.org/docs/latest/install-and-configure/install-opensearch/index/#operating-system-compatibility..

Reply:
Updated the link in my latest commit.